src/pages/admin/store/pickup/index.tsx

```diff
@@ -23,6 +24,14 @@ const AdminPickupPage = ({ futurePickupEvents, pastPickupEvents }: AdminPickupPa
       <div className={styles.header}>
         <Typography variant="h1/bold">Manage Pickup Events</Typography>
 
+        <button
+          type="button"
+          className={`${styles.displayButton} ${display === 'future' && styles.active}`}
+          onClick={() => router.push(`${config.admin.store.pickupCreate}`)}
+        >
+          <Typography variant="h5/bold">Create New Pickup Event</Typography>
+        </button>
```

Comment:
Currently the create button is in the middle of the header, which is a bit weird. I think it makes sense to align it on the right with the `Future | Past` selector

You could do that by removing `justify-content: space-between;` from `.header`, then adding some auto margin between the two elements.

This would work:

```suggestion
        <Typography variant="h1/bold" style={{ marginRight: 'auto' }}>Manage Pickup Events</Typography>

        <button
          type="button"
          className={`${styles.displayButton} ${display === 'future' && styles.active}`}
          onClick={() => router.push(`${config.admin.store.pickupCreate}`)}
        >
          <Typography variant="h5/bold">Create New Pickup Event</Typography>
        </button>
```